server/handlers_v2.go

```diff
@@ -454,9 +454,9 @@ func (server HTTPServer) getImpactedClusters(
 	)
 
 	var aggregatorResp *http.Response = nil
+	var err error
```

Comment:
This function is huge, with many `err`s having different meaning etc. Probably it is a right time to refactor it. Another rule of thumb - is anybody able to describe the function in one simple English sentence?

Reply:
cleaned it up a little bit